$OpenBSD: patch-libbacktrace_backtrace_3,v 1.2 2019/05/20 15:12:28 schwarze Exp $
We renamed the library to execinfo.
Also add the missing function names to the NAME section.
Index: libbacktrace/backtrace.3
--- libbacktrace/backtrace.3.orig
+++ libbacktrace/backtrace.3
@@ -19,10 +19,12 @@
 .Dt BACKTRACE 3
 .Os
 .Sh NAME
-.Nm backtrace
+.Nm backtrace ,
+.Nm backtrace_symbols ,
+.Nm backtrace_symbols_fd
 .Nd glibc and libexecinfo backtrace compatible API
 .Sh SYNOPSIS
-.Fd #include <backtrace.h>
+.Fd #include <execinfo.h>
 .Ft int
 .Fn backtrace "void **buffer" "int depth"
 .Ft char **
